rdkit/Chem/__init__.py

```diff
@@ -127,29 +126,30 @@ def FindMolChiralCenters(mol, force=True, includeUnassigned=False, includeCIP=Tr
     [(2, 'Tet_CCW'), (4, 'Tet_CCW'), (6, 'Tet_CCW')]
 
   """
+  centers: list = []
   if useLegacyImplementation:
     AssignStereochemistry(mol, force=force, flagPossibleStereoCenters=includeUnassigned)
-    centers = []
     for atom in mol.GetAtoms():
       if atom.HasProp('_CIPCode'):
         centers.append((atom.GetIdx(), atom.GetProp('_CIPCode')))
       elif includeUnassigned and atom.HasProp('_ChiralityPossible'):
         centers.append((atom.GetIdx(), '?'))
-  else:
-    centers = []
-    itms = FindPotentialStereo(mol)
-    if includeCIP:
-      atomsToLabel = []
-      bondsToLabel = []
-      for si in itms:
-        if si.type == StereoType.Atom_Tetrahedral:
-          atomsToLabel.append(si.centeredOn)
-        elif si.type == StereoType.Bond_Double:
-          bondsToLabel.append(si.centeredOn)
-      AssignCIPLabels(mol, atomsToLabel=atomsToLabel, bondsToLabel=bondsToLabel)
+    return centers
```

Comment:
!!!!!  This bypasses everything below it. I don't think this is what should happen.

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.